Selenium/exercise10.rb

```diff
@@ -0,0 +1,20 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path="C:/Ruby32-x64/Drivers/chromedriver.exe"
+driver = Selenium::WebDriver.for :chrome
+
+driver.get 'https://computer-database.gatling.io/computers'
+
+table = driver.find_element(:class,'computers')
+header = table.find_element(:tag_name,'thead')
+row = header.find_elements(:tag_name,'th')
+puts row[0].text
+
+body = table.find_element(:tag_name,'tbody')
+col_row = body.find_elements(:tag_name,'tr')
```

Comment:
You do not need to fetch the header row separately. You only need to read the `tr` elements of the table which will also fetch the header values. as they too are enclosed within a `tr` tag.